web-app/src/components/mainLayout/AvatarMenu.vue

```diff
@@ -0,0 +1,59 @@
+<template>
```

Comment:
It is not generic component, i suggest to rename to `HeaderMenu`, what do you think?

Reply:
how about **HeaderAvatarMenu** or **HeaderSettingsMenu** ?

Reply:
`HeaderAvatarMenu` not bad ;)